pkg/controllers/disruption/consolidation_test.go

```diff
@@ -47,6 +47,7 @@ import (
 	"sigs.k8s.io/karpenter/pkg/scheduling"
 	"sigs.k8s.io/karpenter/pkg/test"
 	. "sigs.k8s.io/karpenter/pkg/test/expectations"
+	expectations "github.com/awslabs/operatorpkg/test/expectations"
```

Comment:
We should ideally be able to import this as a dot import, and port whatever expectations are in the `sigs.k8s.io/karpenter` package into operatorpkg. Thoughts on waiting for that to be fixed before merging this?

Reply:
Rather, do you mind removing the function here in sigs/karpenter, and just use the operatorpkg one instead without needing a dot import?